modules/administration-guide/pages/installing-che-on-microsoft-azure.adoc

```diff
@@ -0,0 +1,33 @@
+:_content-type: ASSEMBLY
+:description: Installing {prod-short} on Microsoft Azure
+:keywords: overview, installing-che-on-microsoft-azure
+:navtitle: Installing {prod-short} on Microsoft Azure
+
+[id="installing-{prod-id-short}-on-microsoft-azure"]
+= Installing {prod-short} on Microsoft Azure
```

Comment:
based on the contents of the assembly, would it make sense to change the title to `Installing and setting up {prod-short} on Microsoft Azure`?

Reply:
@deerskindoll 
Yes, it make sense.